.travis.yml

```diff
@@ -9,6 +9,8 @@ bundler_args: --without development
 cache: bundler
 
 before_install:
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install 4
+  - npm install -g yarn
```

Comment:
But I thought this was an opt-in feature and I don't see the `webpacker` gem specified anywhere...

In any case, I still don't understand why we need `nvm` and why we use it to install... nodejs v4. `npm` is available in Travis already, and I'd say that's the only thing you need to install `yarn`.